.changeset/fuzzy-timers-kneel.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris-tokens': minor
+---
+
+Added new z-index tokens that will be the default in v7.0.0 of polaris-tokens
```

Comment:
```suggestion
Added new z-index tokens that will be the default in v7.0.0
```

Light suggestion since this will be in the `polaris-tokens` changelog